projects/igniteui-angular/src/lib/chips/chip.component.html

```diff
@@ -14,27 +14,31 @@
     (over)="onChipOverHandler($event)"
     (dropped)="onChipDrop($event)">
 
-    <div #selectContainer [ngClass]="selectClass(selected)">
-        <ng-container *ngTemplateOutlet="selectIconTemplate"></ng-container>
-    </div>
+    <div class="igx-chip__start" #selectContainer>
+        <div *ngIf="selected" [ngClass]="selectClass(selected)">
```

Comment:
So with this change you are pretty much eliminating the select icon animation. Is this the desired effect?

Reply:
Yes, the animation is eliminated